.pre-commit-config.yaml

```diff
@@ -17,13 +17,11 @@ repos:
     hooks:
       - id: isort
         name: isort (python)
-  - repo: https://github.com/pre-commit/mirrors-yapf
-    rev: v0.32.0
+  - repo: https://github.com/psf/black
+    rev: 23.1.0
     hooks:
-      - id: yapf
-        args: [ "-i", "-r" ]
-        types: [ "python" ]
-        additional_dependencies: [ "toml" ]
+      - id: black
+        exclude: src/scipp/core/cpp_classes.pyi
```

Comment:
Is this not using the `pyproject.toml`?

Reply:
I tried to use `exclude`, `extend_exclude` and `force_exclude` in pyroject.toml. But it seems that pre-commit calls it in such a way that it ignores those settings.